components/rpill/BUILD.gn

```diff
@@ -0,0 +1,15 @@
+import("//brave/components/rpill/buildflags.gni")
+import("//build/config/features.gni")
+
+source_set("rpill") {
+  # Remove when https://github.com/brave/brave-browser/issues/10646 is resolved
+  check_includes = false
+  public_deps = []
+
+  sources = [
+    "browser/rpill.cc",
+    "browser/rpill.h",
+  ]
+
+  deps = []
```

Comment:
you need a public dep for `"//brave/build/rust",` for https://github.com/brave/brave-core/pull/7365/files#diff-c0964d6e042b717dd6d554530a59230cb6db29abaedc3e37b04de0744fba4457R7

Reply:
also the buildflag should be a public dep

Reply:
Not using rust anymore